docs/Style-Guide.md

```diff
@@ -23,132 +22,14 @@ _It is still on you to ensure your code conforms to the `standard` spec, whether
 Since this is a Vue.js project, any pull requests **must** follow *Priority A* rules mentioned in the [Vue.js Style Guide](https://vuejs.org/v2/style-guide/), and *should* follow the *Priority B* rules. Please take the time to read at least those two sections.
 
 ## CSS Style Guide
-We use [Bulma](https://bulma.io/documentation/overview/start/) framework and `SCSS`.
+TODO: complete style guide
 
-### CSS conventions
-- Everything that can be solved with Bulma’s classes should be solved with it.
+Use l- for layout components and c- for anything inside a component. It's just easier to understand where to fine the code and inderstand the dependencies.
+All other classes are globals and should be kept to a minimum for simplicity and avoid interdependencies problems.
```

Comment:
"All other classes are **globals** and should be kept to a minimum for simplicity and avoid **interdependencies** problems." => "All other classes are global and should be kept to a minimum for simplicity and to avoid conflicts."